prime-router/src/main/kotlin/transport/RESTTransport.kt

```diff
@@ -620,6 +620,11 @@ class RESTTransport(private val httpClient: HttpClient? = null) : ITransport {
                             // create JSON object for the BODY. This encodes "/" character as "//", needed for WA to accept as valid JSON
                             JSONObject().put("body", message.toString(Charsets.UTF_8)).toString()
                         }
+                        "application/hl7-v2" -> {
+                            // The following line doesn't work. It shows one seg on their server
+                            val filteredMsg = message.toString(Charsets.UTF_8).replace("\n", "\r").dropLast(1) + "\r"
```

Comment:
Why do you need the `.dropLast(1) + "\r"` part here?

Reply:
Without them, it doesn't work.  I tested with Tomas (Epic engineer).  I think, it removes the EOF character and replace it with /r.